source/beamline/SXRMB/SXRMBBeamline.cpp

```diff
@@ -489,13 +489,21 @@ SXRMBHVControl *SXRMBBeamline::ambiantIC1HVControl() const
 AMAction3* SXRMBBeamline::createBeamOnActions() const
 {
 	if(!beamlineControlShutterSet_->isConnected()) {
-		AMErrorMon::error(this, 0, QString("Failed to create the beam on actions due to unconnected PVs."));
+		AMErrorMon::error(this, 0, QString("Failed to create the beam on actions due to unconnected shutter PVs."), true);
+		return 0;
+	}
+
+	if (SSH1406B1001Shutter_->value() != 1) {
```

Comment:
1 is open. If the safety shutter is not open (2: between, 3: error, 4: closed, ...). We will not allow to turn beam on.